### ISSUE TITLE
Advanced GUI closed/recur

### ISSUE THREAD

**Issue by ctengel** [ctengel/taskmaster#88]
- Also a way to view done tasks? - like #20 and/or #71 ?
- Way to complete a task AND schedule recurrence of it?
  - do a `POST /tasks/<id>/action` close&duplicate
  - distinct from #64 ? (or is this webgui implementation of that CLI version?) 
- Note that delete #46 is not in scope

_Originally posted by @ctengel in https://github.com/ctengel/taskmaster/issues/12#issuecomment-1783885688_

Split from #12 and likely do with/after #11

**Comment by ctengel** [comment]
Show recently closed in main view?
